app/Models/Loggable.php

```diff
@@ -73,16 +73,16 @@ public function logCheckout($note, $target /* What are we checking out to? */)
             'settings' => $settings,
         ];
 
-
-        // Send to the admin, if settings dictate
-        $recipient = new \App\Models\Recipients\AdminRecipient();
-
+        $checkoutClass = null;
```

Comment:
I'm not seeing where this line comes into play.  static::$checkoutClass should be looking for a member variable, correct? Not a variable scoped to this method?  Do we ever use the local $checkoutClass variable?

Reply:
And if it were meant to modify the checkoutClass of the static:: call, this would result in line 79 translating to $target->notify(null) every time, if I understand correctly

Reply:
It's not meant to modify it, just give it a default if we don't have a static declared (as we didn't on components)

Reply:
Where does this get overridden in the file? If I'm reading it correctly it's always set, which means that line 79 always calls static::null, right?

Reply:
static:: overrides based on the static variable set in the model